tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/dynamic_hero_block.html

```diff
@@ -1,17 +1,28 @@
-<div>
-    {% if value.static_text %}<span>{{ value.static_text }}</span>{% endif %}
+<div data-dynamic-hero class="dynamic-hero">
+    {% if value.static_text %}<h2 class="heading heading--three heading--light">{{ value.static_text }}</h2>{% endif %}
     {% if value.dynamic_text %}
         {% if value.dynamic_text|length > 1 %}
-            {# If there's more than one dynamic text, show the controls for the loop. #}
-            <ul>
-                {% for text in value.dynamic_text %}
-                    <li>{{ text }}</li>
-                {% endfor %}
-            </ul>
+            {# If there's more than one dynamic text, show Swiper carousel structure and controls for the loop. #}
+            <div class="swiper">
+                <ul class="swiper-wrapper">
+                    {% for text in value.dynamic_text %}
+                        <li class="swiper-slide">
+                            {# Fixed px values required to control the font size for all browsers and prevent slide overflow#}
+                            <h3 class="heading heading--mega text-[60px] lg:text-[100px]">{{ text }}</h3>
```

Comment:
Question: should we set these in `rem` so they change if the user sets a font size via the browser? Although not sure what knock-on effects changing the font size would have on the carousel.

Reply:
unfortunately, increasing fonts breaks the carousel, with a particularly bad effect on Firefox - that's why fonts are controlled using px utility classes. From an accessibility perspective, 60/100px seems to be large enough though and probably doesn't require further enlargement

![image](https://github.com/user-attachments/assets/5ae10f99-fef3-44b7-a5d2-b587fa145d83)